testdata/baselines/reference/fourslash/hover/JsdocLink1.baseline

```diff
@@ -11,7 +11,7 @@
 //  * @see {@link C} its great
 //  */
 // function CC() {
-//          ^
+//         ^^^
```

Comment:
This seems off - the range includes whitespace. Can you check what the old baseline did here and figure that out?